VAMobile/src/screens/HomeScreen/ProfileScreen/ContactInformationScreen/ContactInformationScreen.tsx

```diff
@@ -132,9 +132,22 @@ function ContactInformationScreen({ navigation }: ContactInformationScreenProps)
     isFetching: loadingContactInformation,
     error: contactInformationError,
     refetch: refetchContactInformation,
+    failureCount,
   } = useContactInformation({ enabled: screenContentAllowed('WG_ContactInformation') })
   const contactInformationInDowntime = useDowntimeByScreenID(ScreenIDTypesConstants.CONTACT_INFORMATION_SCREEN_ID)
   const { contentMarginBottom, gutter, condensedMarginBetween } = theme.dimensions
+  const [retried, setRetried] = useState(false)
+
+  useEffect(() => {
+    if (failureCount > 0) {
+      setRetried(true)
+    }
+
+    if (retried && !loadingContactInformation) {
+      const retryStatus = contactInformationError ? 'fail' : 'success'
+      logAnalyticsEvent(Events.vama_react_query_retry(retryStatus))
+    }
```

Comment:
Is there a reason we couldn't just do if `(failureCount > 0 && !loadingContactInformation)` ?

Reply:
Good question. That's what I wanted to do initially, but after a retry succeeds, the failure count gets wiped, so it wouldn't tell is if the query was retried when logging successful retries